application/controllers/UserManagementController.php

```diff
@@ -1310,29 +1310,19 @@ public function loadModel(int $id): User
     /**
      * Update admin-user
      *
-     * REFACTORED (in UserManagementController)
-     *
      * @param array $aUser array with user details
      * @return object user - updated user object
      * @throws CException
      */
     public function updateAdminUser(array $aUser): User
     {
-        $oUser = User::model()->findByPk($aUser['uid']);
-        //If the user id of the post is spoofed somehow it would be possible to edit superadmin users
-        //Therefore we need to make sure no non-superadmin can modify superadmin accounts
-        //Since this should NEVER be the case without hacking the software, this will silently just do nothing.
```

Comment:
I guess this situations is now talen care on the `userManager->canEdit()` method, right?

Reply:
> I guess this situations is now talen care on the `userManager->canEdit()` method, right?

It's ALL the point : to move to the same function for testing and show … 

and one function use another one …